### PR TITLE
Enable auth presubmit in Prow

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -25,6 +26,59 @@ presubmits:
...
   #                  Otherwise, run a Jenkins job.
 
+  istio/auth:
+  - name: auth-presubmit
+    context: prow/auth-presubmit.sh
+    skip_report: true
+    always_run: true
+    rerun_command: "@istio-testing bazel test this"
+    trigger: "((?m)^@istio-testing (bazel )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
+    branches:
```
Comment (nclando) [inline prow/config.yaml:36]: I think we have decided that we don't need this?

**Review by nlandolfi — APPROVED** [review]
LGTM

File: prow/config.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/config.yaml:36]: I prefer stating which branch this is testing for. We only care about master and release branches and those may have different test requirements.

**Pull request merged**